src/controllers/commands/CommandController.cpp

```diff
@@ -643,7 +643,85 @@ void CommandController::initialize(Settings &, Paths &paths)
             getApp()->windows->getMainWindow().getNotebook().getSelectedPage());
 
         currentPage->getSelectedSplit()->getChannelView().clearMessages();
+        return "";
+    });
 
+    this->registerCommand("/settitle", [](const QStringList &words,
+                                          ChannelPtr channel) {
+        if (words.size() < 2)
+        {
+            channel->addMessage(
+                makeSystemMessage("Usage: /settitle STREAM TITLE."));
+            return "";
+        }
+        if (auto twitchChannel = dynamic_cast<TwitchChannel *>(channel.get()))
+        {
+            auto status = twitchChannel->accessStreamStatus();
+            getHelix()->updateChannel(
+                twitchChannel->roomId(), "", "", words.mid(1).join(" "),
+                [channel](NetworkResult dank) {
+                    channel->addMessage(makeSystemMessage(
+                        QString("Updated title.").arg(dank.status())));
+                },
+                [channel] {
+                    channel->addMessage(
+                        makeSystemMessage("Title update failed! Are you "
+                                          "missing the required scope?"));
+                });
+        }
+        else
+        {
+            channel->addMessage(makeSystemMessage(
+                "Unable to set title of non-Twitch channel."));
+        }
+        return "";
+    });
+    this->registerCommand("/setgame", [](const QStringList &words,
+                                         ChannelPtr channel) {
+        if (words.size() < 2)
+        {
+            channel->addMessage(
+                makeSystemMessage("This command comes from Dankerino patches. "
+                                  "Usage: /setgame STREAM GAME."));
+            return "";
+        }
+        if (auto twitchChannel = dynamic_cast<TwitchChannel *>(channel.get()))
+        {
+            getHelix()->fetchGames(
+                QStringList(), {words.mid(1).join(" ")},
+                [channel, twitchChannel](std::vector<HelixGame> games) {
+                    if (games.size() == 0)
+                    {
+                        channel->addMessage(
+                            makeSystemMessage("Game not found."));
+                    }
+                    else  // 0 or 1 games
+                    {
+                        auto status = twitchChannel->accessStreamStatus();
+                        getHelix()->updateChannel(
+                            twitchChannel->roomId(), games.at(0).id, "", "",
+                            [channel](NetworkResult dank) {
+                                channel->addMessage(
+                                    makeSystemMessage(QString("Updated game.")
```

Comment:
Could we add messaging about which game has been set? if it's available in games.at(0).name

Reply:
Added game name in 516b17ae